### PR TITLE
Update faq to include link to "build containers the hard way"

### PR DESCRIPTION
**Pull request by loosebazooka**
fixes https://github.com/GoogleContainerTools/jib/issues/332

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -479,3 +480,7 @@ To tag the image with a timestamp, simply set the timestamp as the tag for `to.i
 ```groovy
 jib.to.image = 'gcr.io/my-gcp-project/my-app:' + System.nanoTime()
 ```
+
+### Can I learn more about container images?
+
+If you like to learn more about container images [@coollog](https://github.com/coollog) has a guide: [build containers the hard way](https://containers.gitbook.io/build-containers-the-hard-way/) which takes a deep dive into everything involved in getting your code into a container and onto a container registry.
```
Comment (TadCordle) [inline docs/faq.md]:
```suggestion
If you'd like to learn more about container images, [@coollog](https://github.com/coollog) has a guide: [Build Containers the Hard Way](https://containers.gitbook.io/build-containers-the-hard-way/), which takes a deep dive into everything involved in getting your code into a container and onto a container registry.
```

**Comment by loosebazooka** [comment]
Anyone know what's going on with this windows test failure:

```
com.google.cloud.tools.jib.registry.RegistryAuthenticatorTest > testUserAgent FAILED
    java.lang.AssertionError: 
    Expected: a string containing "User-Agent: Competent-Agent"
         but: was ""
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
        at org.junit.Assert.assertThat(Assert.java:956)
        at org.junit.Assert.assertThat(Assert.java:923)
        at com.google.cloud.tools.jib.registry.RegistryAuthenticatorTest.testUserAgent(RegistryAuthenticatorTest.java:194)
```

?? Is this just flaky?

**Comment by TadCordle** [comment]
Ran it again and got a slightly different result

```
com.google.cloud.tools.jib.registry.RegistryAuthenticatorTest > testUserAgent FAILED
    java.lang.AssertionError:
    Expected: a string containing "User-Agent: Competent-Agent"
         but: was "GET /?service=someserver&scope=repository:someimage:pull,push HTTP/1.1
    Accept: */*
    Accept-Encoding:"
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
        at org.junit.Assert.assertThat(Assert.java:956)
        at org.junit.Assert.assertThat(Assert.java:923)
        at com.google.cloud.tools.jib.registry.RegistryAuthenticatorTest.testUserAgent(RegistryAuthenticatorTest.java:194)
```

**Comment by chanseokoh** [comment]
Ah, my bad. That looks flaky on Windows. I'll file an issue, but I suggest we put `@Ignore` for now.

**Comment by TadCordle** [comment]
We can merge this if we update the branch with @chanseokoh's fix

**Comment by loosebazooka** [comment]
manually rebased against master, hopefully windows tests are passing now.

**Pull request merged**